api/src/main/resources/config/application.properties

```diff
@@ -48,6 +48,7 @@ app.baseDomain=osmt.wgu.edu
 app.baseUrl=https://${app.baseDomain}
 app.frontendUrl=https://${app.baseDomain}
 app.loginSuccessRedirectUrl=${app.frontendUrl}/login/success
+spring.jackson.serialization.FAIL_ON_EMPTY_BEANS=false
```

Comment:
What cases were you encountering with empty beans?

Reply:
oops that config was for testing purposes, we can safely remove that config